pkg/runtime/wrapper.go

```diff
@@ -39,6 +43,61 @@ CMD [%s]
 ENTRYPOINT ["/bin/membrane"]
 `
 
+const wrapperDockerFileWithOTel = `
+ARG BASE_IMAGE
+
+FROM ${BASE_IMAGE}
+
+RUN apk update \
+  &&   apk add --no-cache ca-certificates wget shadow \
+  &&   update-ca-certificates
```

Comment:
Is the base image always alpine?

Reply:
I have moved these changes into the base images